test/api.js

```diff
@@ -0,0 +1,262 @@
+'use strict';
+var path = require('path');
+var test = require('tap').test;
+var Api = require('../api');
+
+test('ES2015 support', function (t) {
+	t.plan(1);
+
+	var api = new Api([path.join(__dirname, 'fixture/es2015.js')]);
+
+	api.run()
+		.then(function () {
+			t.same(api.passCount, 1);
+		});
+});
+
+test('generators support', function (t) {
+	t.plan(1);
+
+	var api = new Api([path.join(__dirname, 'fixture/generators.js')]);
+
+	api.run()
+		.then(function () {
+			t.same(api.passCount, 1);
+		});
+});
+
+test('async/await support', function (t) {
+	t.plan(1);
+
+	var api = new Api([path.join(__dirname, 'fixture/async-await.js')]);
+
+	api.run()
+		.then(function () {
+			t.same(api.passCount, 2);
+		});
+});
+
+test('display filename prefixes for failed test stack traces', function (t) {
+	t.plan(3);
+
+	var files = [
+		path.join(__dirname, 'fixture/es2015.js'),
+		path.join(__dirname, 'fixture/one-pass-one-fail.js')
+	];
+
+	var api = new Api(files);
+
+	api.run()
+		.then(function () {
+			t.same(api.passCount, 2);
+			t.same(api.failCount, 1);
+			t.match(api.errors[0].title, /test (›|») fixture (›|») one-pass-one-fail (›|») this is a failing test/);
```

Comment:
Instead of `(›|»)`, you should use: `figures.pointerSmall` (and require `figures` module at the top).